scripts/python/release_label_validator.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import os
+import yaml
+import sys
+
+with open('.github/release.yml', 'r') as file:
+    try:
+        release_yaml = yaml.safe_load(file)
+        label_pool = []
+        for category in release_yaml['changelog']['categories']:
+            label_pool.extend(category['labels'])
+    except yaml.YAMLError as exc:
+        print(exc)
+
+print(f"Label pool: {label_pool}")
+token = os.getenv('GITHUB_TOKEN')
+repo = os.getenv('REPOSITORY')
+
+response = requests.get(f'https://api.github.com/repos/{repo}/releases/latest', headers={'Authorization': f'token {token}'})
+response.raise_for_status()
+latest_release = response.json()
+latest_release_date = latest_release['created_at']
+
+response = requests.get(f'https://api.github.com/repos/{repo}/pulls?state=closed&sort=updated&direction=desc', headers={'Authorization': f'token {token}'})
+response.raise_for_status()
+all_closed_prs = response.json()
+
+prs_since_last_release = [
+    pr for pr in all_closed_prs
+    if pr['merged_at'] is not None and pr['merged_at'] > latest_release_date
```

Comment:
How about these merged after the release was created but not taken by release.
I mean is it possible that merged_at > latest_release_date but the point in time when we created tag and cut off some PRs to be released is after latest_release_date.